experimental/handoff-library/csharp_dotnetcore/Microsoft.Bot.Builder.Handoff/IHandoffRequest.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Schema;
+
+namespace Microsoft.Bot.Builder.Handoff
+{
+    /// <summary>
+    /// The interface allows callers to track the completion of the handoff request.
+    /// </summary>
+    public interface IHandoffRequest
+    {
```

Comment:
WE've larglgley gotten away from Interfaces, as they are impossible to version. 

If (later) we add another method to this interface, we lose all compatibility. 

Better to have this be a base class if possible, so we can add / change in a standard compatible way. 

(This compat issue is directly addressed in .Net Core 3, but we're not there yet.....)